pr47-core/src/vm/al31f/analysis/anf.rs

```diff
@@ -0,0 +1,107 @@
+
+pub type DefineSymbol = String; // todo
+pub type LocalSymbol = String; // todo
+pub type SymbolRef = String; // todo
+
+
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Do not derive everywhere. use `cfg_attr` when some derivation is for debug purpose.